docs/metrics/getting-started/README.md

```diff
@@ -0,0 +1,59 @@
+# Getting Started with OpenTelemetry .NET in 5 Minutes
+
+First, download and install the [.NET Core
+SDK](https://dotnet.microsoft.com/download) on your computer.
+
+Create a new console application and run it:
+
+```sh
+dotnet new console --output getting-started
+cd getting-started
+dotnet run
+```
+
+You should see the following output:
+
+```text
+Hello World!
+```
+
+Install the
+[OpenTelemetry](../../../src/OpenTelemetry/README.md)
+package:
+
+```sh
+dotnet add package OpenTelemetry
+```
+
+Update the `Program.cs` file with the code from [Program.cs](./Program.cs):
+
+Run the application again (using `dotnet run`) and you should see the metric
+output from the console, similar to shown below:
+
+<!-- markdownlint-disable MD013 -->
+```text
+Export[] 14:54:56.162 14:54:57.155 TestMeter:counter [tag1=value1;tag2=value2] SumMetricAggregator Value: 15977610, Details: Delta=True,Mon=True,Count=1597761,Sum=15977610
```

Comment:
nit: probably not related to this PR, "Mon" could be misleading and we might want "Monotonic".

"Delta=True" - consider something like "Temporality=Delta" (and "Temporality=Cumulative").

Reply:
agree. to be addressed when exporter is properly written inside Console exporter package.